bot/exts/moderation/voice_gate.py

```diff
@@ -160,6 +186,35 @@ async def on_message(self, message: discord.Message) -> None:
         with suppress(discord.NotFound):
             await message.delete()
 
+    @Cog.listener()
+    async def on_voice_state_update(self, member: Member, *_) -> None:
+        """Pings a user if they've never joined the voice chat before and aren't verified."""
```

Comment:
I don't care too much, but I don't see a reason to obfuscate the other params. I think for clarity's sake it's better to show what we receive, even if we don't use it. Later on, it may cause someone to build a needlessly complex solution to a problem that can be solved by simply using the params.

Reply:
Should I correct it in the other places in the file as well?  The `voice_verify` command has that as well.

Reply:
If you want. :smile: I wouldn't ask for it. This comment is opinionated anyway, if you prefer it with the `*_` I will live.

Reply:
Oh, it's different for a command. I imagine the command has it so that you can do `!voice_verify a b c d`. For the listener, we know exactly what we will receive, so we can annotate the params.

Reply:
Changed in the latest commit.